docs/users/detailed-guides/data-sources/git-repositories.md

```
@@ -1,661 +1,188 @@
-
-   ```bash
-   # Test manual clone
-   git clone https://github.com/org/repo.git /tmp/test-clone
-   ```
-
-3. **Verify token permissions**:
-   - For GitHub: Ensure token has `repo` scope for private repos
-   - For GitLab: Ensure token has `read_repository` scope
-
+1. **Verify repository URL**: - Ensure the URL is correct and includes `.git` extension - Check if the repository is public or private - Verify you have access to the repository
+2. **Check authentication**: ```bash # Test manual clone git clone https://github.com/org/repo.git /tmp/test-clone ```
```

markdownlint-cli2 (0.17.2)

Spaces inside code span elements

(MD038, no-space-in-code)